modules/completion/selectrum/TODO.org

```diff
@@ -26,8 +26,10 @@ https://github.com/minad/marginalia/issues/59
 - org (check, might be fine)
 - counsel-minibuffer-history analogue
 - counsel-company analogue
-- bibtex
 * PROJ Other
 ** TODO bind =consult-lsp-diagnostics= to something?
-** TODO test out bibtex-actions, check if more configuration should be added
-https://github.com/bdarcus/bibtex-actions
+** TODO bibtex-actions improvements?
+currently =SPC n b= is bound to a function, but =bibtex-actions= doesn't have a
+main dispatch function like =ivy-bibtex=, rather it has a bunch of different
+ones. Binding the ~bibtex-actions-map~ there would probably be better, but there
```

Comment:
On the map, I mostly included that for `embark` integration.

But I did raise this issue, which is broader than this package, on the `embark` issue tracker:

https://github.com/oantolin/embark/issues/197

Reply:
this seems to be a different thing. The issue here is that doom binds the keybinding (and all the other ones) before `bibtex-actions` is loaded, and there doesn't seem to a way to autoload a keymap as far as i can tell. This isn't an issue with putting this on your private config since that loads later